src/comp_loinc/schema/code_schema.yaml

```diff
@@ -16,21 +16,21 @@ classes:
 
   LoincCodeClass:
     is_a: Thing
-    slots:
-      - loinc_number
-      - long_common_name
-      - formal_name
-      - short_name
-      - status
-      - loinc_class
-      - loinc_class_type
-
-      - has_component
-      - has_property
-      - has_time
-      - has_system
-      - has_scale
-      - has_method
+    slots:  # First 2 blocks: LoincTableCore.csv | Last 2 blocks: LoincPartLink_Supplementary.csv
```

Comment:
## Code schema comments
- [ ] Make sure these comments are correct

I'm just trying to understand how the LinkML model relates to the inputs.

What is `formal_name`? I don't see this defined anywhere in the LOINC release files.
Is this like the fully specified name?: `<component/analyte>:<kind of property>:<time aspect>:<system type>:<scale>:<method>`?

<details><summary>Screenshot</summary>
<p>

<img width="1167" alt="loinc table core" src="https://github.com/user-attachments/assets/e010201e-240e-4251-a761-1f22fd83c150">

</p>
</details>